package.json

```diff
@@ -18,7 +18,10 @@
     "blakejs": "^1.0.1",
     "blue-tape": "^1.0.0",
     "browserify": "^16.5.1",
-    "node-fetch": "^2.6.0",
+    "eventsource": "^1.1.0",
+    "jest": "^27.4.3",
```

Comment:
I think, npm allows to distinguish between prod and test dependencies. Though not sure.

Reply:
I moved test dependencies into a "devDependencies" section. This stackoverflow answer goes into the difference sections: https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies